### PR TITLE
Update the documentation

### PR DESCRIPTION
**Pull request by cwognum**
## Changelogs

- `BaseAction` classes now reference the docs of the respective functional API.
- Move docs from `Field(description=...)` to regular docstrings.

---

_Checklist:_

- [X] _Was this PR discussed in an issue? It is recommended to first discuss a new feature into a GitHub issue before opening a PR._
- [ ] ~_Add tests to cover the fixed bug(s) or the newly introduced feature(s) (if appropriate)._~
- [X] _Update the API documentation if a new function is added, or an existing one is deleted._
- [X] _Write concise and explanatory changelogs above._
- [X] _If possible, assign one of the following labels to the PR: `feature`, `fix` or `test` (or ask a maintainer to do it for you)._

---

### CODE REVIEW

File: auroris/cli.py [hunk 1]
```diff
@@ -11,17 +11,24 @@
...
+    if dataset_path is not None:
+        curator.src_dataset_path = dataset_path
+
     # Run curation
-    dataset, report = curator(dataset)
+    dataset, report = curator.transform()
 
     # Save dataset
     dm.fs.mkdir(destination, exist_ok=overwrite)
     path = dm.fs.join(destination, "curated.csv")
```
Comment (zhu0619) [inline auroris/cli.py:26]: The file extension needs to be updated.

File: auroris/curation/actions/_deduplicate.py [hunk 2]
```diff
@@ -54,20 +53,18 @@ def deduplicate(
 class Deduplication(BaseAction):
     """
     Automatic detection of outliers.
+
+    See [`auroris.curation.functional.deduplicate`][] for the docs of the
```
Comment (zhu0619) [inline auroris/curation/actions/_deduplicate.py:57]: > [`auroris.curation.functional.deduplicate`][]
Is the second brackets on purpose? hyperlink?

**Review by zhu0619 — APPROVED** [review]
Thanks @cwognum. 
Just one minor thing to change.

File: auroris/curation/actions/_deduplicate.py (hunk 2 above)
Comment (cwognum) [reply auroris/curation/actions/_deduplicate.py:57]: This is the syntax to reference code in `mkdocs`. It's short for: 

```
[`auroris.curation.functional.deduplicate`][auroris.curation.functional.deduplicate]
```

File: auroris/cli.py (hunk 1 above)
Comment (cwognum) [reply auroris/cli.py:26]: Good catch!

**Pull request merged**